.env.sample

```diff
@@ -6,4 +6,7 @@ A_PASSWORD="<priv_password>"
 B_EMAIL="<reg_email>"
 B_PASSWORD="<reg_password>"
 
-CATEGORY="all"
\ No newline at end of file
+CATEGORY="plugin"
```

Comment:
Suggestion: Either comment showing all category settings or where categories are listed and if possible explained.

Reply:
Additionally: A_EMAIL and B_EMAIL might best start out as EMAIL or when we can clearly name the user types: PATRON_EMAIL, ARCHIVE_EMAIL, ADMIN_EMAIL